source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -0,0 +1,162 @@
+title: Read the Locally-Managed Master Key from a File
+ref: read-local-master-key-from-file
+level: 4
+content: |
+  First, retrieve the contents of the local master key file that you generated
+  in the :ref:`Create a Master Key <fle-create-a-master-key>` section:
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        .. code-block:: java
+           :emphasize-lines: 11
+
+           String path = "master-key.txt";
+
+           byte[] fileBytes = new byte[96];
+
+           try (FileInputStream fis = new FileInputStream(path)) {
+             fileBytes = fis.readAllBytes();
+           } catch (IOException e) {
+             e.printStackTrace();
+           }
+
+           final byte[] localMasterKey = Arrays.copyOf(fileBytes, 96);
+---
+title: Specify KMS Provider Settings
+ref: specify-kms-provider-settings
+level: 4
+content: |
+  Next, specify the KMS provider settings. The client uses these settings to
+  discover the master key. Set the provider name to ``local`` when using a local
+  master key:
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        The KMS provider settings are stored in a Map in order to use the
+        :java-docs-latest:`kmsProviders helper
+        method <javadoc/com/mongodb/ClientEncryptionSettings.Builder.html#kmsProviders(java.util.Map)>
```

Comment:
Missing terminating ` on this line reslts broken link.

Reply:
Good catch, thanks.